l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -498,13 +498,13 @@ def setDatiGeneraliDocumento(self, invoice, body):
                 if not causale:
                     continue
                 causale_list_200 = \
-                    [causale[i:i+200] for i in range(0, len(causale), 200)]
+                    [causale[i:i + 200] for i in range(0, len(causale), 200)]
                 for causale200 in causale_list_200:
                     # Remove non latin chars, but go back to unicode string,
                     # as expected by String200LatinType
                     causale = causale200.encode(
                         'latin', 'ignore').decode('latin')
-                    body.DatiGenerali.DatiGeneraliDocumento.Causale\
+                    body.DatiGenerali.DatiGeneraliDocumento.Causale \
```

Comment:
penso che lo spazio in fondo sia di troppo.

Reply:
ok, rimosso e lasciato solo modifica del replace